packages/e2e-tests/specs/editor/various/inserter.test.js

```diff
@@ -20,4 +24,27 @@ describe( 'Inserter', () => {
 		const isPreviewVisible = await preview.isIntersectingViewport();
 		expect( isPreviewVisible ).toBe( true );
 	} );
+
+	it( 'last-inserted block should be given and keep the focus', async () => {
+		await page.focus( '.block-editor-default-block-appender__content' );
+		await page.keyboard.type( 'Testing inserted block focus' );
```

Comment:
You can contract these into one command :slightly_smiling_face: 

```suggestion
		await page.type(
			'.block-editor-default-block-appender__content',
			'Testing inserted block focus'
		);
```